### ISSUE TITLE
Exercise to use opencv

### ISSUE THREAD

**Issue by gimseng** [gimseng/99-ML-Learning-Projects#42]

**Comment by wkostuch** [comment]
What exactly do you want done here?  Just some examples using openCV?

**Comment by pragyakapoor** [comment]
Hey, maybe I can help with this if you provide some more details about what exactly are you looking for.

**Comment by gimseng** [comment]
Hi @wkostuch and @pragyakapoor , I only had a vague idea of one/a few projects primarily using openCV. I'd imagine the more basic/simple the better for beginners. If there are a few projects you have mind, we can split into a few project folders as well. Its entirely up to you, but I would suggest 1-2 beginner projects to teach someone to get used to using openCV.

**Comment by pragyakapoor** [comment]
@gimseng great, I have two mini projects in mind that are simple to follow and gives a good idea about openCV to a beginner. Gonna start working on it asap.

**Comment by gimseng** [comment]
@pragyakapoor perfect ! Feel free to create PR/issue tickets as needed, and do let us know if you have any questions after reading the guidelines and look at a few previous project to get an idea of the format. If you need more help, feel free to ask here or in the discord.

**Comment by tanviagwl98** [comment]
I too have a basic project with the implementation of opencv . Can I go ahead and create a pull request?

**Comment by gimseng** [comment]
Hi @tanviagwl98 thanks for the interest. What data and models do you have in mind? 
@pragyakapoor could you tell us more on the project, what data and models you are using? If both of you have different examples, then its great to have both contributions ( and PRs ). If there's overlap, maybe we can discuss further here how to collaborate.

**Comment by tanviagwl98** [comment]
@gimseng I have idea of "Cartoonization of Images" using opencv and numpy with some filter and edge detection. In this project There is need of picture to perform cartoon editing using python script. Please let me know if I should go further with this or not?

**Comment by pragyakapoor** [comment]
@tanviagwl98 I have done a similar thing which was live sketching and it
uses edge detection as well. So, maybe you could work on something
different?

**Comment by tanviagwl98** [comment]
I will go through your project @pragyakapoor if it will reflect similar to yours I will try something else.
But I think there may be differences and help beginners understand efficiently if both are posted.

**Comment by Dhruv-VINT** [comment]
I can add a fun OpenCV project for this issue ! @gimseng you must have heard about that Invisibility cloak project, it's super easy and thoroughly implements OpenCV. What do you think?

**Comment by gimseng** [comment]
Hi @Dhruv-VINT, that sounds like a fun project. I am not very familiar with openCV, though played around with #91, but do make sure that one can run it on google colab. I seem to have trouble with #91 in getting it to turn on my camera or something. 

Thanks !

**Comment by thenorthkun** [comment]
Yes, Although my project is also implemented on a local system but I'll try to figure it out how to run it on google colab.

**Comment by prenoyk** [comment]
@gimseng Is this still open? If it is I have a project applies watermark using LSB method

**Comment by Moddy2024** [comment]
@gimseng  Hi I want to contribute to this repo for hacktoberfest 2022. If you have something specific in mind you can tell me that or I will choose some good projects using OpenCV on my own.

**Comment by VivekMarwal** [comment]
Hi @gimseng, I can add some good example project.  I am working with Infosys & Comcast on RDK-V project. So i can add some example.
